src/evaluators.py

```diff
@@ -0,0 +1,68 @@
+""" Module for defining the different types of evalulators that can be
+    applied to a list of signals. """
+
+class Evaluator:
+    """ Parent Evaluator that filters signals """
+    def __init__(self, threshold):
+        self.threshold = threshold
+
+    def evaluate_value(self, value):
+        return None
+
+    def evaluate_signals(self, detected_signals):
+        confirmed_signals = {}
+
+        for detected_signal in detected_signals:
+            value = detected_signal.interval
+            if self.evaluate_value(value):
+                confirmed_signals[detected_signal] = value
+
+        return confirmed_signals
+                
+""" Children of Evaluator """
```

Comment:
This seems unnecessary.

Reply:
My point was not the class is unnecessary. Just the string before it.

Reply:
AH. Ok, agreed.